### PR TITLE
Transactions

### PR DESCRIPTION
**Pull request by kensteph**
In this PR I : 
✅ Add a column species of type string to the animals' table.
✅ Add data to the animals' table.
✅ Use transaction to Update, Delete data from the animals' table.
✅ Query displaying how many animals.
✅ Query displaying how many animals never tried to escape.
✅ Query displaying the average weight of animals.
✅ Query displaying who escapes the most.
✅ Query displaying the minimum and maximum weight of each type of animal.
✅ Query displaying the average number of escape attempts of each type of animal.

Here is the [Screenshot](https://drive.google.com/file/d/1_ZBwlKdmka8FrszLQd-GXw0i8YZsWv-M/view?usp=sharing) .

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -16,3 +17,58 @@
 	SELECT * FROM animals WHERE name !='Gabumon';
 -- Find all animals with a weight between 10.4kg and 17.3kg (including the animals with the weights that equals precisely 10.4kg or 17.3kg)
 	SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
+
+-- ====================================== Part 2 ==================================================================
+-- Inside a transaction update the animals table by setting the species column to unspecified. Verify that change was made. 
+-- Then roll back the change and verify that the species columns went back to the state before the transaction.
+BEGIN;
+UPDATE animals SET species = 'unspecified';
+ROLLBACK;
```
Comment (mikethreels) [inline queries.sql]: - [ ] according to the requirements you should Verify that the change was made. Then roll back the change and verify that the species columns went back to the state before the transaction.

File: queries.sql [hunk 2]
```diff
@@ -16,3 +17,58 @@
...
+-- Inside a transaction:
+-- Update the animals table by setting the species column to digimon for all animals that have a name ending in mon.
+-- Update the animals table by setting the species column to pokemon for all animals that don't have species already set.
+-- Commit the transaction.
+-- Verify that change was made and persists after commit.
+BEGIN;
+UPDATE animals SET species = 'digimon' WHERE name LIKE '%mon';
+UPDATE animals SET species = 'pokemon' WHERE species IS NULL;
+COMMIT;
+
```
Comment (mikethreels) [inline queries.sql]: - [ ] according to the requirements you should Verify that the change was made and persists after the commit.

File: queries.sql [hunk 3]
```diff
@@ -16,3 +17,58 @@
...
+-- Now, take a deep breath and... Inside a transaction delete all records in the animals table, then roll back the transaction.
+BEGIN;
+DELETE FROM animals;
+ROLLBACK;
+
```
Comment (mikethreels) [inline queries.sql]: - [ ] according to the requirements After the rollback you should verify if all records in the animals table still exist.

**Review by mikethreels — CHANGES_REQUESTED** [review]
Hi Ken, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- linters are passing

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by kensteph** [comment]
Thank you @mikethreels for your review.

**Review by m05e5 — APPROVED** [review]
**Hi @kensteph 
You did great. Your project is complete!  There is nothing else to say other than... it's time to merge:shipit: 
Congratulations! 🎉 
CleverScornfulBuffalo-size_restricted

## Highlights

✔️ Professional readme file
✔️ Clean Codebase
✔️ No linters error
✔️ Great job implementing the requested changes

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

_**m05e5**_

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._**

**Pull request merged**